static/docs/tutorials/deep/define-ml-pipeline.md

```diff
@@ -6,24 +6,28 @@ To include a data file into your data science environment, you need to copy the
 file into the repository. We'll create a `data/` directory for the data files
 and download a 40MB data archive into this directory.
 
-<details>
-
-### Expand to learn how to download on Windows
-
-Windows doesn't include the `wget` utility by default, so you'll need to use the
-browser to download `data.xml`. Save it into the `data/` subdirectory.
-(Right-click [this link](https://data.dvc.org/tutorial/ver/data.zip) and click
-`Save Link As...`(Chrome).
-
-</details>
-
 ```dvc
 $ mkdir data
 $ wget -P data https://data.dvc.org/tutorial/nlp/100K/Posts.xml.zip
 $ du -sh data/*
  41M data/Posts.xml.zip
 ```
 
+<details>
```

Comment:
can we do the same for the get started - wrap the same info into expandable section?

Reply:
I'm not following. The only reference to downloading with Windows in the get-started is in https://dvc.org/doc/get-started/connect-code-and-data, already collapsed in the **Expand to prepare example code** block.

Reply:
ah, I see. My bad. It is indeed a part of the larger expandable section.